DOKS-CI-CD/README.md

```diff
@@ -1135,46 +1109,31 @@ Explanations for the `DOKS-CI-CD/assets/manifests/tekton/` folder structure:
   - `tekton-argocd-build-deploy-trigger-template.yaml` - contains the definition for the Tekton TriggerTemplate used this tutorial (in-depth explanations can be found inside).
 - `kustomization.yaml` - this is the main kustomization file (in-depth explanations can be found inside).
 
-**Important Note:**
-
-- Please bear in mind that the `configs` folder used by Kustomize contains **sensitive data**. So, make sure to not commit those files in your Git repository (you can use a `.gitignore` file for this purpose).
+**Note:** The `configs` folder used by Kustomize contains sensitive data. Use a `.gitignore` file to exclude commiting those files in your Git repository.
 
-Please follow below steps to create all required resources (Tekton CRDs, Knative CRDs, etc) in your Kubernetes cluster, via Kustomize:
+Create all required resources in your Kubernetes cluster, via Kustomize. Edit and save each property file from the `DOKS-CI-CD/assets/manifests/tekton/configs` subfolder, making sure to replace the `<>` placeholders accordingly. For example, you can use [VS Code](https://code.visualstudio.com):
 
-1. Clone the `container-blueprints` repo (if not already):
+```shell
+code DOKS-CI-CD/assets/manifests/tekton/configs/argocd/auth.env
 
-    ```shell
-    git clone https://github.com/digitalocean/container-blueprints.git
-    ```
+code DOKS-CI-CD/assets/manifests/tekton/configs/docker/config.json
 
-2. Change directory to your local copy:
+code DOKS-CI-CD/assets/manifests/tekton/configs/github/pat.env
+```
 
-    ```shell
-    cd container-blueprints
-    ```
+  **Hints:**
 
-3. Edit and save each property file from the `DOKS-CI-CD/assets/manifests/tekton/configs` subfolder, making sure to replace the `<>` placeholders accordingly. For example you can use [VS Code](https://code.visualstudio.com):
+  - To obtain Argo CD admin password, use the following command:
 
     ```shell
-    code DOKS-CI-CD/assets/manifests/tekton/configs/argocd/auth.env
-
-    code DOKS-CI-CD/assets/manifests/tekton/configs/docker/config.json
-
-    code DOKS-CI-CD/assets/manifests/tekton/configs/github/pat.env
+    kubectl -n argocd get secret argocd-initial-admin-secret -o jsonpath="{.data.password}" | base64 -d
```

Comment:
While we are here can you change this shell snippet to:

```
kubectl -n argocd get secret argocd-initial-admin-secret -o jsonpath="{.data.password}" | base64 -d; echo
```

Avoids issues with password copy because of the missing new line.